nilearn/_utils/data_gen.py

```diff
@@ -797,8 +799,9 @@ def create_fake_bids_dataset(base_dir='',
                              with_confounds=True,
                              confounds_tag="desc-confounds_timeseries",
                              no_session=False,
-                             random_state=0):
-    """Creates a fake :term:`bids<BIDS>` dataset directory with dummy files.
+                             random_state=0,
+                             entities=None,):
```

Comment:
TODO: some some input validation to ensure that all labels associated with an entity are alphanumeric.

Reply:
done